tests/unit/conftest.py

```diff
@@ -1,6 +1,7 @@
 """A place to put fixture functions that are useful application-wide."""
 import functools
 
+import pytest
```

Comment:
Hokay. This file already contained an `@pytest.fixture` but it was not importing `pytest`, and yet it worked? :raised_eyebrow: There's an `import *` in this file that I suspect must have been importing `pytest`. Best to have it here explicitly I think (and we should fix that `import *`, but that's a separate issue)